CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 **Added**:
 
+- **decidim-core**: Add a searchable users endpoint to the GraphQL api. [\#5474](https://github.com/decidim/decidim/pull/5474)
```

Comment:
Add also a changelog entry for the autocomplete of mentions please :smile:

Reply:
pst, it was a secret! 😅